src/main/java/eu/ha3/presencefootsteps/sound/SoundEngine.java

```diff
@@ -174,7 +173,8 @@ public Identifier getFabricId() {
     @Override
     public CompletableFuture<Void> reload(Synchronizer sync, ResourceManager sender,
             Profiler serverProfiler, Profiler clientProfiler,
-            Executor serverExecutor, Executor clientExecutor) {
+            Executor serverExecutor, Executor clientExecutor)
+    {
```

Comment:
Brace should be on the same line, please, and restore the terminating newline at the end of this file.